SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -201,22 +201,29 @@ void USpatialConnectionManager::Connect(bool bInitAsClient, uint32 PlayInEditorI
 
 void USpatialConnectionManager::OnLoginTokens(void* UserData, const Worker_LoginTokensResponse* LoginTokens)
 {
+	USpatialConnectionManager* ConnectionManager = static_cast<USpatialConnectionManager*>(UserData);
```

Comment:
It might be preferable to use an unreal cast here (Cast) just as we are handling an unreal type (UObject)

Reply:
You can't really do that with a `void*` though - UE `Cast<T>` only work with `UObjects`.

Reply:
This is true!